src/styles/card.scss

```diff
@@ -0,0 +1,127 @@
+.products {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the value '1' is redundant in the 'repeat' function. Use '200px' instead of 'repeat(1, 200px)'.